integration_tests/pom.xml

```diff
@@ -344,6 +344,23 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
This is a copy of the parent pom.xml file https://github.com/NVIDIA/spark-rapids/blob/branch-22.04/pom.xml#L782-L797,

Changed nothing but overwrite config: `<detectOfflineLinks>false</detectOfflineLinks>`